documentation/modules/ROOT/pages/04-quarkus-extensions.adoc

```diff
@@ -758,49 +719,40 @@ The REST call about should return an response like:
 }
 ----
 
+Then you can stop the container by typing kbd:[Ctrl+C].
+
 [#qext-add-custom-probes-to-k8s]
 === Add probes to Kubernetes Quarkus Application
 
-Before we add the probes to the Kubernetes application, we need to push the rebuilt containers to the container registry.
+We need to push the rebuilt containers to the container registry.
 
 include::ROOT:partial$build-containers.adoc[tag=tag-push]
 
-Once the rebuilt containers has been successfully pushed to the container registry run the following command to add the probes to the kubernetes deployment.
+[#qext-custom-health-check-k8s-delete-resources]
+=== Delete Kubernetes Resources
 
-ifndef::workshop[]
-[tabs]
-====
-kubectl::
-+
---
-[#qext-copy-k8s-custom-probes-file]
-[source,bash,subs="+macros,+attributes"]
-----
-cp pass:[$TUTORIAL_HOME]/templates/custom-probes.yaml pass:[$TUTORIAL_HOME]/work/{quarkus-project-name}/custom-probes.yaml
-----
-copyToClipboard::qext-copy-k8s-custom-probes-file[]
+include::ROOT:partial$k8s-build-deploy.adoc[tag=k8s-delete]
 
-Apply the probes to the existing `pass:[$APP_NAME]` Kubernetes deployment:
+[#qext-custom-health-check-k8s-apply-resources]
+=== Apply Kubernetes Resources
 
-[#qext-k8s-set-cusotm-probes]
-[source,bash,subs="+macros,+attributes"]
-----
-kubectl patch deployment pass:[$APP_NAME] -p "$(cat pass:[$TUTORIAL_HOME]/work/{quarkus-project-name}/custom-probes.yaml)"
-----
-copyToClipboard::qext-k8s-set-custom-probes[]
---
-oc::
-+
---
-endif::[]
-[#qext-oc-set-custom-probes]
+Since we are using Kubernetes extension, configured at xref:03-containers-kubernetes.adoc[Cloud Native Deployment] section, the kubernetes resource has been updated with both probes.
+
+[#quarkusk8s-generated-kubernetes-resource-custom-health-check]
 [source,bash,subs="+macros,+attributes"]
 ----
-oc set probe deployment pass:[$APP_NAME] --liveness --get-url=http://:8080/health/live
-oc set probe deployment pass:[$APP_NAME] --readiness --get-url=http://:8080/health/ready
+cat pass:[$TUTORIAL_HOME]/target/wiring-classes/META-INF/kubernetes/kubernetes.json
```

Comment:
same comment as above lets use k8s for all minikube types and openshift yaml/json for all workshop. As using openshift yaml allows us to create route automatically.  If not we need to add instruction to `expose svc`

Reply:
We are adding this expose svc thing in the invoke section.